test/jest/LndClient.spec.ts

```diff
@@ -5,6 +5,14 @@ import { getValidDeal } from '../utils';
 import { SwapRole } from '../../lib/constants/enums';
 import { ClientStatus } from '../../lib/swaps/SwapClient';
 
+const getOpenChannelSyncResponse = () => {
```

Comment:
Also minor but does this need to be a function? It can just be an object with `hasFundingTxidStr` and `getFundingTxidStr` I think, it confused me a bit as currently used.

Reply:
I actually wanted to keep same structure as `getSendPaymentSyncResponse` but I rather to keep as object too :)